content/package.json

```diff
@@ -10,13 +10,15 @@
         "url": "https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components"
     },
     "scripts": {
+        "aemfed": "aemfed -t \"http://admin:admin@localhost:4502\" -e \"**/*___jb_(old|tmp)___\" -w \"src/content/jcr_root/\"",
         "eslint": "eslint .",
```

Comment:
Any chance we can turn host,port and user/pass into start parameters ?

Reply:
@msagolj - I've removed the target definition in c0faaec58d2ea5026ef9bd90d54353fc2b56c902, it's the default. If you need a specific target you can run for example:

`npm run aemfed -- -t "http://admin:admin@localhost:8080"`